clients/mobile/src/setup/notification/saveDeviceToken.ts

```diff
@@ -24,3 +32,23 @@ export const saveDeviceTokenToSNS = async(
     )
   }
 }
+
+export const saveDeviceTokenLocally = async(
+  deviceToken: string
+): Promise<void> => {
+  const loggedInUser = await authService.currentLogInUser()
+  await StorageService.storeItem<{ deviceToken: string; userId: string }>(
+    TOKEN.DEVICE_TOKEN,
+    { userId: loggedInUser.userId, deviceToken }
+  )
+}
+
+export const checkDeviceAlreadyRegisteredForSameUser = async(): Promise<boolean> => {
```

Comment:
this name is also a bit strange
`isDeviceAlreadyRegisteredForUser`
Maybe it should be passed current `loggedInUser.userId`, to make it simple.

Then, you can do something like


```
if (!(await isDeviceAlreadyRegisteredForUser(deviceId, userId))) {
// do something
return
}
return 
```

Reply:
resolved in [9a0e8d2a6](https://github.com/CraftsmenLtd/BloodConnect/pull/272/commits/9a0e8d2a659996bbf0a9183fdf1252e7f074ca9f)